utils/bump_cwl_version.py

```diff
@@ -13,74 +15,22 @@
 ERROR_MISSING_DOCKER_REQUIREMENT = 'CWL tool is missing DockerRequirement'
 ERROR_MISSING_DOCKER_PULL = 'Please specify "dockerPull" in your DockerRequirement and rerun script'
 
+
 def tools_list(tools_dir):
```

Comment:
Could you update the function and argument to use `templates_*` instead of `tools_*` since that's what it's listing now?